tests/Support/MailFakeTest.php

```diff
@@ -11,7 +12,7 @@ public function setUp(): void
         parent::setUp();
 
         App::shouldReceive('getLocale')->andReturn('en/US');
-        Mail::swap(new MailFake());
+        Mail::swap(new MailFake(new MailManager(app())));
```

Comment:
Would this need to use `$this-app` to ensure that it is the test-constructed app?

```suggestion
        Mail::swap(new MailFake(new MailManager($this->app)));
```

Reply:
I´ve changed it. I guess that both apps are the same, but this makes it more clear which one is being used.